### PR TITLE
fix: login by pressing enter key

### PR DESCRIPTION
**Pull request by kibo-amolpawar**

### CODE REVIEW

File: components/layout/Login/LoginContent/LoginContent.spec.tsx [hunk 1]
```diff
@@ -68,41 +75,50 @@ describe('[components] (LoginContent)', () => {
...
+
+const loginInputs = async (user: UserEvent) => {
+  const emailInput = screen.getByRole('textbox', { name: 'email' })
+  const passwordInput = screen.getByLabelText('password')
+
+  await user.type(emailInput, 'example@example.com')
+  await user.type(passwordInput, 'abc')
+
+  await user.tab()
+}
```
Comment (kibo-sushant) [inline components/layout/Login/LoginContent/LoginContent.spec.tsx:124]: Some of the places test cases name are not meaning full, please correct it something like below

should show entered email input => should show user entered email 
should show entered password input => should show user entered password
should disable login button when user enters invalid inputs  => should keep login button disable when user enters invalid credentials 
should enable login button when user enters valid inputs and  call onLoginMock when user click onLogin  => should enable login button and call onLoginMock when user enters valid credentials and clicks on Login button 
should call onForgotPasswordClickMock when click forgotPasswordLink => should call onForgotPasswordClickMock callback function when user clicks on forgotPasswordLink
should login when user entered valid credentials & enter key pressed => should login when user enters valid credentials and press enter key

File: components/layout/Login/LoginContent/LoginContent.spec.tsx [hunk 2]
```diff
@@ -68,41 +75,50 @@ describe('[components] (LoginContent)', () => {
...
+    await loginInputs(user)
 
+    const loginButton = screen.getByRole('button', { name: 'common:log-in' })
     await waitFor(() => expect(loginButton).toBeEnabled())
-    await act(async () => {
-      await user.click(loginButton)
-    })
+
+    await user.click(loginButton)
     await waitFor(() => expect(onLoginMock).toHaveBeenCalled())
```
[the remark this answers is not in the document]
Comment (amolpawar-kibo) [reply components/layout/Login/LoginContent/LoginContent.spec.tsx]: Done.

File: components/layout/Login/LoginContent/LoginContent.spec.tsx (hunk 1 above)
Comment (amolpawar-kibo) [reply components/layout/Login/LoginContent/LoginContent.spec.tsx:132]: Done.

File: components/layout/Login/LoginContent/LoginContent.tsx [hunk 3]
```diff
@@ -95,72 +95,75 @@ const LoginContent = (props: LoginContentProps) => {
...
-              ref={null}
-              required
-              sx={{ ...styles.formInput }}
-              onBlur={field.onBlur}
-              onChange={(_name, value) => field.onChange(value)}
-              error={!!errors?.email}
-              helperText={errors?.email?.message}
-              autoFocus={true}
-            />
+      <Box component="form" onSubmit={handleSubmit(handleLogin)} id="loginForm">
```
[the remark this answers is not in the document]
Comment (amolpawar-kibo) [reply components/layout/Login/LoginContent/LoginContent.tsx]: Done.

**Review by TriptiVermaXP — APPROVED** [review]

**Review by kibo-kevinwatts — APPROVED** [review]

**Review by sushantj2050dev — APPROVED** [review]

**Pull request merged**